src/v3/test/integration/flow-unlock-account-email-polling-to-mfa-authenticator.test.tsx

```diff
@@ -61,15 +62,19 @@ describe('flow-unlock-account-email-polling-to-mfa-challenge-authenticator', ()
       return Promise.reject(new Error('Unknown request'));
     });
     const {
-      container, findByText,
+      container, findByText, findByTestId,
     } = await setup({ mockRequestClient });
 
     await findByText(/Verify with your email/);
 
     /* refresh: 4000 and we poll twice */
-    jest.advanceTimersByTime(9000);
+    jest.advanceTimersByTime(1000);
```

Comment:
Is the comment wrong or is the synthetic clock advancement wrong?

Reply:
The comment is right, I just increased it by 1 second to make sure it doesn't miss timing or anything weird happens. 1 second should have been enough though, just my paranoia made me increase it